### PR TITLE
test(frontend): use absolute path for src imports

### PR DESCRIPTION
**Pull request by ktmud**
### CATEGORY

- [x] Refactor
- [x] Build / Development Environment

### SUMMARY

This changes all JS imports in `specs` to use absolute path, which makes the code a little bit easier to read and shall make rearranging the test files easier.

Added a `jsconfig.json` for VS Code so that intellisense can still work.

Also upgraded `prettier` to fix lint.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TEST PLAN

Make sure CI passes and other IDEs also have no problem using JS features.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS

@kristw @rusackas @villebro

### CODE REVIEW

File: superset-frontend/.prettierrc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "arrowParens": "avoid"
```
Comment (ktmud) [inline superset-frontend/.prettierrc:4]: `prettier` changed the default behavior for `arrowParent`. We revert it back to the old behavior.

Comment (kristw) [reply superset-frontend/.prettierrc:4]: There seems to be another change to space between `function` and parentheses.

`function() {}` vs `function () {}`

I do not have strong preference in terms of styling but perhaps we can use the original to reduce amount of changes. 

This is the settings from `@superset-ui`

```ts
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "printWidth": 100,
  "proseWrap": "always",
  "requirePragma": false,
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false
};
```

Comment (ktmud) [reply superset-frontend/.prettierrc:4]: https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot

It seems Prettier folks decided this is somehow the preferred style and didn't add an option to turn it off.

Comment (kristw) [reply superset-frontend/.prettierrc:4]: ic. If no strong objection from others I am good with this. I'll approve but wait for others to see a bit.

**Review by kristw — APPROVED** [review]

File: superset-frontend/.prettierrc (hunk 1 above)
Comment (rusackas) [reply superset-frontend/.prettierrc:4]: I prefer without the space, but if there's no way to turn it off, it won't break my heart.

Comment (rusackas) [reply superset-frontend/.prettierrc:4]: I just read through some of the threads on that feature, and now I'm glad I don't work on that project. Yeesh!

Comment (ktmud) [reply superset-frontend/.prettierrc:4]: Can we merge this then? 😄

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
Jeez... I keep spending forever looking at these things, and then when I'm done, I look up and see `Merged` 🤣Again, this looks great, and thanks.

**Pull request merged**